servers/services/src/main/java/org/projectnessie/services/rest/TreeResource.java

```diff
@@ -335,6 +336,50 @@ public EntriesResponse getEntries(String namedRef, EntriesParams params)
     }
   }
 
+  @Override
+  public EntriesResponse getNamespaceEntries(
+      String refName,
+      @Nullable String hashOnRef,
+      @Nullable String namespacePrefix,
+      @Nullable Integer depth)
+      throws NessieNotFoundException {
+    WithHash<NamedRef> refWithHash = namedRefWithHashOrThrow(refName, hashOnRef);
+    try {
+      List<EntriesResponse.Entry> entries;
+      try (Stream<EntriesResponse.Entry> s =
```

Comment:
Would it make sense to change `s` to a more meaningful name, let's say `stream`? Not a big deal though

Reply:
done